src/screens/WalletConnect/CallRequest/TransactionRequestContent.js

```diff
@@ -132,22 +138,24 @@ const useTransactionFee = (request: WalletConnectCallRequest) => {
   const fee = BigNumber(getFormattedTransactionFeeValue(feeInWei ?? '', feeInfo?.gasToken)) || null;
   const gasSymbol = feeInfo?.gasToken?.symbol || ETH;
 
-  const balances = useRootSelector(accountEthereumWalletAssetsBalancesSelector);
-  const { amount, symbol, decimals } = useTransactionPayload(request);
-
   const chain = chainFromChainId[request.chainId];
   if (!chain && !estimationErrorMessage) {
     estimationErrorMessage = t('error.walletConnect.cannotDetermineEthereumChain');
   }
 
+  const accountAssetsBalances = useRootSelector(accountAssetsBalancesSelector);
```

Comment:
Not related to this particular line. I thing we should put `TransactionDeploymentWarning` here to signal that accepting this transaction will deploy your wallet.

Reply:
Additionally added Archanova account check to `TransactionDeploymentWarning` since we shouldn't show this warning for Archanova accounts.